src/Microsoft.Sbom.Common/IFileSystemUtils.cs

```diff
@@ -65,5 +65,10 @@ public interface IFileSystemUtils
         /// Returns whether a directory is empty.
         /// </summary>
         bool IsDirectoryEmpty(string directoryPath);
+
+        /// <summary>
+        /// RandomTempPath to be used in scenarios where a (-m) and (-di) are provided and the user does not provide a (-b) path.
+        /// </summary>
+        public static string RandomTempPath = Path.Combine(Path.GetTempPath(), Path.GetRandomFileName());
```

Comment:
I would prefer not to store static value in the interface. Can you change this interface to declare method instead, implement this method on fileSystemUtils classes that we have, call it early when initializing config, and then pass it around in config object rather than referencing this value from interface in many places?

Reply:
Interfaces should only declare behaviors and should very rarely contain any implementations (ideally never).

Reply:
Fixed in new commit.